src/Repository/ProductCommentRepository.php

```diff
@@ -54,21 +77,58 @@ class ProductCommentRepository
     const DEFAULT_COMMENTS_PER_PAGE = 5;
 
     /**
+     * @param ManagerRegistry $registry
      * @param Connection $connection
      * @param string $databasePrefix
      * @param bool $guestCommentsAllowed
      * @param int $commentsMinimalTime
      */
     public function __construct(
-        Connection $connection,
+        $registry,
+        $connection,
         $databasePrefix,
         $guestCommentsAllowed,
         $commentsMinimalTime
     ) {
+        parent::__construct($registry, ProductComment::class);
         $this->connection = $connection;
         $this->databasePrefix = $databasePrefix;
         $this->guestCommentsAllowed = (bool) $guestCommentsAllowed;
         $this->commentsMinimalTime = (int) $commentsMinimalTime;
+        /* Only works since PS 8.0.0 - Doctrine\Cache 1.11.x
+        $cachePool = new FilesystemAdapter();
+        $cache = DoctrineProvider::wrap($cachePool);
+        $config = $this->connection->getConfiguration();
+        $config->setResultCacheImpl($cache);
+        */
```

Comment:
I think you could simply remove this code if it is not used.

```suggestion
```

Reply:
It is a plan-for-future code. I don't care much of storing it explicitly in Release code, or storing it implicitly in github history.

Reply:
Do you know the concept of [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it) 😛 ?

What happens quite often with plan-for-future code is that we _think_ we're going to use it... but then we do something else 😅 and a few months later, another developer comes, he opens the code and he is confused by the code he sees. He does not know what to do with this commented code: is it important? should he follow it? is it something from the past?

This is why a lot of developers avoid commented out code https://methodpoet.com/commented-out-code/#:~:text=What%20is%20the%20commented%2Dout,issues%20if%20it's%20not%20removed.

Can I suggest to rather store this into a commit of yours, in git history, but do not put it in the release code? 😉

Reply:
Yep, the plan-for-future code was wiped out from the Release code.